### PR TITLE
+ new function "temporarily enable/disable style..."

### PR DESCRIPTION
**Pull request by kongomongo**
Thought it would be quite nice to be able to temporarily disable a style without deleting it or permanently disable respective styles in options.

### CODE REVIEW

File: stylebot/editor/style.js [hunk 1]
```diff
@@ -616,13 +623,20 @@ stylebot.style = {
    * @param {String} css The CSS to apply to the page.
    */
   resetPreview: function() {
+    // needsReset: only reset stuff that really needs a reset. This is to prevent the resetPreview() fired from port.onDisconnect() to override the enable/disable style toggle and the PreviewPopover
+    if (this.needsReset) {
+      this.needsReset = false;
+
+      if (this.status) {
```
Comment (wmertens) [inline stylebot/editor/style.js]: Shouldn't you indent the rest of the block?

Comment (kongomongo) [reply stylebot/editor/style.js]: indeed

**Comment by ankit** [comment]
i included this feature in the current release in the store. thank you

**Comment by kongomongo** [comment]
awesome thank you!

**Pull request closed without merging**